src/applications/personalization/profile360/containers/VAProfileApp.jsx

```diff
@@ -21,11 +21,8 @@ class VAProfileApp extends React.Component {
     return (
       <div>
         <RequiredLoginView
-          authRequired={1}
           serviceRequired={backendServices.USER_PROFILE}
           user={this.props.user}
-          loginUrl={this.props.loginUrl}
```

Comment:
No longer supported by `RequiredLoginView`. These were removed here https://github.com/department-of-veterans-affairs/vets-website/pull/6961 and here https://github.com/department-of-veterans-affairs/vets-website/pull/6474